.github/workflows/dockerByPR.yml

```diff
@@ -0,0 +1,85 @@
+name: DockerCI-PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+    paths-ignore:
+      - 'docs/**'
+      - '.github/**'
```

Comment:
Didn't notice this in the other reviews (for java and typescript) but will probably have to go back and update this list. 

This should include I think all of the md files, can you do *.md?

Reply:
Fixed with commit 5f40b20